src/Tests/InstrEngineTests/InstrEngineTests/RefStructsTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace InstrEngineTests
+{
+#if NET7_0_OR_GREATER
```

Comment:
>NET7_0_OR_GREATER [](http://example.com/codeflow?start=4&length=17)

Is there a way that we can put this condition in the csproj instead? Just exclude these files from the build?

Reply:
Looks like you can do:

```xml
<ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">
  <Compile Remove="RefStructsTests.cs" />
<ItemGroup>
```